### DISCUSSION TITLE
Sending logs from release builds

### DISCUSSION THREAD

**Discussion by cparish312** [jasonjmcghee/rem#101] [General]
Is there any way for a user to send us logs if users are going from a released build or is the best way to debug for them to build themselves and see the logs?

**Comment by jasonjmcghee** [comment]
Haven't set any of this up and have no experience in this department (proper way to do logging in native Mac apps), fwiw, but the non-mac way to do this would be to just log to disk in a logs folder.

**Comment by dollarusername** [comment]
Logging to just something under /var/log would be great!
